pyproject.toml

```diff
@@ -15,7 +15,7 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.7"
-click = "^7.1"
+click = "^8.0.1"
```

Comment:
Does this break backwards-compatibility with older versions of Nautobot that use click 7.x? Might need to relax this constraint to permit both versions.

Reply:
Good Call, updated